### PR TITLE
Update cssparser to 0.9

### PR DESCRIPTION
**Pull request by SimonSapin**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15552)

### CODE REVIEW

**Comment by SimonSapin** [comment]
Tidy fails because of `[replace]`: https://github.com/servo/servo/pull/15555

**Comment by SimonSapin** [comment]
This is now ready to land. r? @Manishearth

File: tests/unit/stylo/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ doctest = false
 [dependencies]
 app_units = "0.3"
 atomic_refcell = "0.1"
-cssparser = {version = "0.8"}
+cssparser = {version = "0.9"}
```
Comment (nox) [inline tests/unit/stylo/Cargo.toml]: Super-nit: `"0.9"` instead of a dict.

**Review by nox — APPROVED** [review]
r=me with that one super nit.

File: tests/unit/stylo/Cargo.toml (hunk 1 above)
Comment (SimonSapin) [inline tests/unit/stylo/Cargo.toml]: Done.

**Pull request merged**